### PR TITLE
Update option's hypertext ref breadcrumbsExample

### PR DESCRIPTION
**Pull request by mmaous**
#### Fixes #0000
- There's a bug whenever you try testing (clicking breadcrumbs) the demo it keeps redirecting the user to the home page!

### CODE REVIEW

**Comment by palantirtech** [comment]
Thanks for your interest in [palantir/blueprint](https://github.com/palantir/blueprint), @mmaous! Before we can accept your pull request, you need to sign our contributor license agreement - just visit https://cla.palantir.com/ and follow the instructions. Once you sign, I'll automatically update this pull request.

File: packages/docs-app/src/examples/core-examples/breadcrumbsExample.tsx [hunk 1]
```diff
@@ -48,13 +48,13 @@ const ITEMS: BreadcrumbProps[] = [
     { icon: "folder-close", text: "All files" },
     { icon: "folder-close", text: "Users" },
     { icon: "folder-close", text: "Janet" },
-    { href: "#", icon: "folder-close", text: "Photos" },
-    { href: "#", icon: "folder-close", text: "Wednesday" },
+    { href: "https://blueprintjs.com/docs/#core/components/overflow-list#", icon: "folder-close", text: "Photos" },
```
Comment (adidahiya) [inline packages/docs-app/src/examples/core-examples/breadcrumbsExample.tsx]: this page may be served on different url paths (for example localhost or a /versions/4/ folder for different major versions), so let's drop everything before the `#`:

```suggestion
    { href: "#core/components/overflow-list", icon: "folder-close", text: "Photos" },
```

File: packages/docs-app/src/examples/core-examples/breadcrumbsExample.tsx [hunk 2]
```diff
@@ -48,13 +48,13 @@ const ITEMS: BreadcrumbProps[] = [
...
+    { href: "https://blueprintjs.com/docs/#core/components/overflow-list#", icon: "folder-close", text: "Wednesday" },
```
Comment (adidahiya) [inline packages/docs-app/src/examples/core-examples/breadcrumbsExample.tsx]:
```suggestion
    { href: "#core/components/overflow-list", icon: "folder-close", text: "Wednesday" },
```

File: packages/docs-app/src/examples/core-examples/breadcrumbsExample.tsx [hunk 3]
```diff
@@ -48,13 +48,13 @@ const ITEMS: BreadcrumbProps[] = [
...
     { icon: "document", text: "image.jpg", current: true },
 ];
 // Show less items for always redner example so we can see when everything fits
 const ITEMS_FOR_ALWAYS_RENDER: BreadcrumbProps[] = [
-    { href: "#", icon: "folder-close", text: "Root" },
+    { href: "https://blueprintjs.com/docs/#core/components/overflow-list#", icon: "folder-close", text: "Root" },
```
Comment (adidahiya) [inline packages/docs-app/src/examples/core-examples/breadcrumbsExample.tsx]:
```suggestion
    { href: "#core/components/overflow-list", icon: "folder-close", text: "Root" },
```

**Review by adidahiya — CHANGES_REQUESTED** [review]

File: packages/docs-app/src/examples/core-examples/breadcrumbsExample.tsx [hunk 4]
```diff
@@ -48,13 +48,13 @@ const ITEMS: BreadcrumbProps[] = [
     { icon: "folder-close", text: "All files" },
     { icon: "folder-close", text: "Users" },
     { icon: "folder-close", text: "Janet" },
-    { href: "#", icon: "folder-close", text: "Photos" },
-    { href: "#", icon: "folder-close", text: "Wednesday" },
+    { href: "#core/components/overflow-list", icon: "folder-close", text: "Photos" },
```
Comment (adidahiya) [inline packages/docs-app/src/examples/core-examples/breadcrumbsExample.tsx:51]: actually, this component is used at both `#core/components/overflow-list` and `#core/components/breadcrumbs`. so we'll need a different solution that links to the current page in each usage of `<BreadcrumbsExample>`.

**Review by adidahiya — CHANGES_REQUESTED** [review]

**Pull request closed without merging**